scripts/bash/buildInternalDependencies.sh

```diff
@@ -52,6 +56,10 @@ if [[ $watch == "true" ]]; then
     echo "yarn concurrently ${build_commands[@]}"
     eval "yarn concurrently ${build_commands[@]}"
 else
+    echo "Deleting existing internal dependency builds"
+    echo "${delete_command}"
+    eval "${delete_command}"
```

Comment:
@rohan-bes I reverted my changes in the [Fix babel caching commit](https://github.com/beyondessential/tupaia/pull/1729/commits/2e06531ba05255f4ca153e855a361aec4d776aef). In that commit, we delete the `dist` folder before building a package. While this works in actual deployments, `yarn build-internal-dependencies` doesn't work in watch mode: I suspect this is because watch mode expects a `dist` folder to pre-exist

@edmofro Tagging you here cause I think you should also have a look at this

Reply:
By the way, I created a test feature deployment using this branch it seems to be working fine

Reply:
Also, deleted files will still not be picked up in `watch` mode. This is the existing behaviour, which can result in weird scenarios during local development: eg

1. A dev deletes a file in a dependency. There forget to delete an `import` of this file
2. The change is not being picked up. However, the dev assumes that hot reload would have picked all changes
3. Dev tests their code locally, everything looks good, so they commit & push their change. They haven't run `build-internal-dependencies` since they pulled their base branch - theoretically, they shouldn't have to
4. They deploy their code and it doesn't work

But at least now deployment builds map accurately to the repository code.

Reply:
This lgtm @kael89, thanks for trying out that undocumented babel command, it seemed good, but this also seems a solid strategy. I'm not worries about that scenario you mentioned above (and nor is it a regression)